CI-CD/GitHub/github-action-gradle.yml

```diff
@@ -0,0 +1,46 @@
+name: Mend Unified Agent Scan - Android
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: set up JDK 8
+      uses: actions/setup-java@v2
+      with:
+        java-version: '8'
+        distribution: 'adopt'
+
+    - name: Setup Android SDK
+      uses: android-actions/setup-android@v2
+
+    - name: Grant execute permission for gradlew
+      run: |
+        chmod +x gradlew
+        echo "ANDROID_HOME:" $ANDROID_HOME
+    - name: Build with Gradle
+      run: ./gradlew build
+
+    - name: Mend Unified Agent Scan
+      env:
```

Comment:
@lukebrogan-mend I believe Gradle scans should also have fileSystemScan disabled

Reply:
added